torch/onnx/_internal/exporter.py

```diff
@@ -875,6 +875,14 @@ def pre_export_passes(
             diagnostic_context, module, original_model
         ).run()
 
+    # This operation should be invoked as the last pre export pass.
```

Comment:
Should this note be in the class docstring as well?

Reply:
good idea